### ISSUE TITLE
No scrollbar

### ISSUE THREAD

**Issue by pootzko** [influxdata/docs.influxdata.com#86]
Hi,

not sure if this is by design or if it is a bug but the new docs page lost it's sidebar for the main content as of today. If by design - please rethink this as it's making it very hard and slow to navigate longer pages.

Thanks

**Comment by rossmcdonald** [comment]
@pootzko Can you attach a screenshot of what you're seeing? That may have been unintentional...

**Comment by pootzko** [comment]
Sure, here:

![sidebar-missing](https://cloud.githubusercontent.com/assets/535842/11795021/4d19462c-a2ac-11e5-8572-6579bc1297ec.png)

The navigation scrollbar is there, but the content one is missing (not even the default chrome sidebar is being displayed). I have been using the docs quite a bit the last few days and the sidebar was there. This morning I noticed it was missing. The green part of the screenshot is the desktop background.

I now checked firefox as well and the problem does not seem to be manifesting there. I have the most recent version of both FF and Chrome. Windows 8.1. Other websites show the sidebar normally.
